scheduler/scd_utils.py

```diff
@@ -73,6 +73,8 @@ def check_line(self, yyyymmdd, hhmm, experiment, scheduling_mode, prio, duration
         :type   duration:           str
         :param  kwargs_string:      kwargs for the experiment instantiation. (Default value = '')
         :type   kwargs_string:      str
+        :param  embargo:            flag for embargoing files. (Default value = '')
+        :type   embargo:            str
```

Comment:
Why is the value a string, if embargo is just a flag? (Just curious, not critiquing the code)

Reply:
Hahaha this reads like you're trying not to set me off. 

I've changed it to a bool, this reads clunky and passing around boolean values is much nicer to handle.